src/test/java/uk/gov/hmcts/reform/sscs/helper/SscsHelperTest.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.reform.sscs.helper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.NO;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.YesNo.YES;
+import static uk.gov.hmcts.reform.sscs.helper.SscsHelper.updateDirectionDueDateByAnAmountOfDays;
+
+import java.time.LocalDate;
+import java.util.Arrays;
+import junitparams.JUnitParamsRunner;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import uk.gov.hmcts.reform.sscs.ccd.domain.*;
+
+@RunWith(JUnitParamsRunner.class)
+public class SscsHelperTest {
+
+    private SscsCaseData sscsCaseData;
+
+    @Before
+    public void setUp() {
+        sscsCaseData = SscsCaseData.builder().ccdCaseId("ccdId").build();
+        Appeal appeal = Appeal.builder()
+                .benefitType(BenefitType.builder().code(Benefit.CHILD_SUPPORT.getShortName()).build()).build();
+        sscsCaseData.setAppeal(appeal);
+    }
+
+    @Test
+    public void givenNoResponseDueDate_WhenOtherPartyIsAdded_ThenSetResponseDueDateTo14DaysInTheFuture() {
+        sscsCaseData.setDirectionDueDate(null);
+        sscsCaseData.setOtherParties(Arrays.asList(buildOtherParty("No", null), buildOtherParty("Yes", NO)));
+
+        updateDirectionDueDateByAnAmountOfDays(sscsCaseData);
+        assertThat(sscsCaseData.getDirectionDueDate()).isEqualTo(LocalDate.now().plusDays(14).toString());
```

Comment:
Due to it's ubiquity in this test class, you could save LocalDate.now() to a global variable today.